js/views/chatConversationVw.js

```diff
@@ -0,0 +1,165 @@
+var Backbone = require('backbone'),
+  $ = require('jquery'),
+  moment = require('moment'),
+  app = require('../App.js').getApp(),
+  loadTemplate = require('../utils/loadTemplate'),
+  baseVw = require('./baseVw'),
+  ChatMessageVw = require('./chatMessageVw');
+
+module.exports = baseVw.extend({
+  className: 'chatConversation',
+
+  events: {
+    'click .js-closeConversation': 'onClickClose',
+    'keydown textarea': 'onKeydownMessage',
+    'keyup textarea': 'onKeyupMessage',
+    'click .js-conversationSettings': 'toggleConvoSettings',
+    'click .chatConversationMenu a': 'closeConvoSettings',
+    'click .js-blockUser a': 'onBlockClick'
+  },
+
+  initialize: function(options) {
+    this.options = options || {};
+
+    if (!options.model) {
+      throw new Error('Please provide a model.');
+    }
+
+    if (!options.user) {
+      throw new Error('Please provide the model of the logged-in user.');
+    }
+
+    if (!options.collection) {
+      throw new Error('Please provide a collection.');
+    }
+
+    this.user = this.options.user;
+
+    this.collection.fetch({
+      data: {
+        guid: this.model.get('guid')
+      },
+      reset: true
+    });
+
+    this.listenTo(this.collection, 'reset', () => {
+      this.renderMessages();
```

Comment:
No, I want renderMessages() to be called either way, since it will hide the loading spinner. It uses some forEach's to iterate through the messages. If there are no messages, it just won't iterate.